client/verta/tests/unit_tests/conftest.py

```diff
@@ -74,5 +74,5 @@ def __repr__(self):  # avoid network calls when displaying test results
     return MockRegisteredModelVersion(
         mock_conn,
         mock_config,
-        _RegistryService.ModelVersion(id=555),
+        _RegistryService.ModelVersion(id=555, registered_model_id=123)
```

Comment:
This is used in the call to registry that now occurs (see [comment below](https://github.com/VertaAI/modeldb/pull/3785/files#r1183118508))